jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -107,19 +107,26 @@
             <action id="lambda.function.gotohandler"
                     class="software.aws.toolkits.jetbrains.services.lambda.actions.GoToHandlerAction"/>
             <separator/>
-            <action id="lambda.function.edit"
-                    class="software.aws.toolkits.jetbrains.services.lambda.actions.EditFunctionAction"/>
+            <action id="lambda.function.updateConfiguration"
+                    class="software.aws.toolkits.jetbrains.services.lambda.actions.UpdateFunctionConfigurationAction"/>
+            <action id="lambda.function.updateCode" class="software.aws.toolkits.jetbrains.services.lambda.actions.UpdateFunctionCodeAction"/>
             <action id="lambda.function.delete"
                     class="software.aws.toolkits.jetbrains.services.lambda.actions.DeleteFunctionAction"/>
         </group>
+        <group id="aws.toolkit.explorer.lambda.stackFunction" popup="true" compact="false">
+            <reference ref="RunContextGroupInner"/>
+            <separator/>
+            <action id="lambda.stackFunction.gotohandler" class="software.aws.toolkits.jetbrains.services.lambda.actions.GoToHandlerAction"/>
+        </group>
         <group id="aws.toolkit.explorer.lambda.application" popup="true" compact="false">
             <action id="lambda.application.delete"
                     class="software.aws.toolkits.jetbrains.services.lambda.applications.DeleteApplicationAction"/>
         </group>
         <action class="software.aws.toolkits.jetbrains.core.SettingsSelectorAction" id="aws.settings.selector" icon="AwsIcons.Logos.AWS"/>
         <action class="software.aws.toolkits.jetbrains.core.credentials.CreateOrUpdateCredentialProfilesAction" id="aws.settings.upsertCredentials" icon="AwsIcons.Logos.AWS"/>
 
-        <group id="aws.toolkit.serverless.template">
+        <!-- Uncomment when 'feature/deploy-application' is merged -->
+        <!-- group id="aws.toolkit.serverless.template">
```

Comment:
Keep this uncommented in destination branch

Reply:
Added a commit to get branch building again